### ISSUE TITLE
[*] dpkg-deb not found, downloading. [-] dpkg download URL is not reachable. Status code: 404 error when trying to run python main.py for the first time

### ISSUE THREAD

**Issue by 0n** [itsnebulalol/permasigner#43]
## What is the expected behavior?
opens permasigner
## What is the actual behavior?
IPA Permasigner | Version main-72710b5
Program created by Nebula | Original scripts created by zhuowei | CoreTrust bypass by Linus Henze

[*] dpkg-deb not found, downloading.
[-] dpkg download URL is not reachable. Status code: 404

## What are the steps to reproduce this bug?
install this on a brand new install of Manjaro after installing python-pip

## Please provide debug logs:
Use the `--debug` option while running the script, and paste the output **into the codeblocks**.

```
IPA Permasigner | Version main-72710b5
Program created by Nebula | Original scripts created by zhuowei | CoreTrust bypass by Linus Henze

[DEBUG] Checking if command ldid is in PATH...
[DEBUG] Checking ldid output...
[DEBUG] On Linux x86_64, ldid not found...
[DEBUG] Checking ldid hash...
[DEBUG] ldid hash successfully verified.
[DEBUG] Checking if command dpkg-deb is in PATH...
[DEBUG] On Linux x86_64, dpkg-deb not found...
[*] dpkg-deb not found, downloading.
[DEBUG] Downloading dpkg-deb on Linux x86_64.
[-] dpkg download URL is not reachable. Status code: 404

```

## Any possible solutions?
no idea, I guess reupload that file on another server

## Can you identify the location in the source code where the problem exists?
no

## Version Information
- iOS Version: irrelevant, my phone isn't even plugged in
- Jailbreak Name and Version: irrelevant, my phone isn't even plugged in

**Comment by FreQRiDeR** [comment]
Manjaro is ARCH based. You need Debian based linux distro to be able to install dpkg.

**Comment by itsnebulalol** [comment]
Fixed in #44

**Comment by superrnovae** [comment]
> Manjaro is ARCH based and uses RPM. You need Debian based linux distro to be able to run dpkg.

It was failing to download because dpkg got updated. dpkg-deb can be used on any distro.
